lib/modules/manager/types.ts

```diff
@@ -174,6 +174,7 @@ export interface PackageDependency<T = Record<string, any>> extends Package<T> {
   editFile?: string;
   separateMinorPatch?: boolean;
   extractVersion?: string;
+  parents?: string[];
```

Comment:
```suggestion
```

Move to `NpmManagerData`

https://github.com/renovatebot/renovate/blob/03b28828f3a4f27476ae7d3959e5c74fe7948442/lib/modules/manager/npm/types.ts#L73-L78